src/components/AppNavigation/ViewButtons.vue

```diff
@@ -1,11 +1,60 @@
-<template>
+<!--
+  - @copyright Copyright (c) 2018 Georg Ehrke <oc.list@georgehrke.com>
+  -
+  - @author Georg Ehrke <oc.list@georgehrke.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
 
+<template>
+	<div class="togglebuttons">
+		<button :class="{active: (selectedView === 'agendaDay')}" class="button first" @click="view('agendaDay')">{{ labelAgendaDay }}</button>
+		<button :class="{active: (selectedView === 'agendaWeek')}" class="button middle" @click="view('agendaWeek')">{{ labelAgendaWeek }}</button>
+		<button :class="{active: (selectedView === 'month')}" class="button last" @click="view('month')">{{ labelMonth }}</button>
+	</div>
 </template>
 
 <script>
-    export default {
-        name: "ViewButtons"
-    }
+export default {
+	name: 'TodayButton',
+	computed: {
+		labelAgendaDay() {
+			return t('calendar', 'Day')
```

Comment:
What do you think of a basic translation component ?

Reply:
cc @skjnldsv ^

Reply:
Well, we have the translate function used all over nextcloud and linked to the transifex component. I think it's working really well, so I don't think changing it is a good idea. It's easier to have the same solution all over nextcloud :)